openpmd_viewer/openpmd_timeseries/data_reader/h5py_reader/params_reader.py

```diff
@@ -212,12 +212,15 @@ def simplify_record(record_comps):
     if 'momentum/x' in record_comps:
         record_comps.remove('momentum/x')
         record_comps.append('ux')
```

Comment:
One could even go so far to only add "u(x|y|z)" aliases if the ED-PIC extension is set in the data series.